tt_metal/hw/ckernels/wormhole_b0/metal/llk_api/llk_sfpu/ckernel_sfpu_ceil.h

```diff
@@ -0,0 +1,40 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "ckernel.h"
+#include "ckernel_defs.h"
+#include "sfpi.h"
+#include "noc_nonblocking_api.h"
+
+using namespace sfpi;
+
+namespace ckernel {
+namespace sfpu {
+
+template <bool APPROXIMATION_MODE, int ITERATIONS = 8>
+inline void calculate_ceil()
+{
+    for (int d = 0; d < ITERATIONS; d++)
+    {
+        vFloat result = dst_reg[0];
+        vFloat v = result;
+        vInt tmp = float_to_int16(result);
```

Comment:
Can you leave a TODO comment here that once float to int32 cast is added, this implementation should be fixed to use that? Ill open an issue for this case for all the ops that require it.